scripts/check-links.mjs

```diff
@@ -6,6 +6,10 @@ import { remark } from "remark";
 import reporter from "vfile-reporter";
 import { visit } from "unist-util-visit";
 
+// Some URLs are valid (e.g. they link to marketing sites or docs that are not hosted through clerk-docs) so they should be excluded from the check.
+// These URLs will be used with the .startsWith() method, so they should be specific enough to not match any URLs that should be checked.
+const EXCLUDE_LIST = ['/pricing', '/docs/reference/backend-api', '/docs/reference/frontend-api']
```

Comment:
We have https://clerk.com/docs/reference/frontend-api already so I added it, even though it's not used in any mdx files yet afaik